package.json

```diff
@@ -27,16 +27,16 @@
     "db:migrate-up-all": "npm run db:migrate",
     "db:migrate-down": "./src/server/database/migrate.sh down",
     "db:migrate-down-all": "./src/server/database/migrate.sh down-all",
-    "lint": "eslint 'src/**/*.js' 'tests/*.js' 'webpack/*.js'",
+    "lint": "eslint --fix 'src/**/*.js' 'tests/*.js' 'webpack/*.js'",
```

Comment:
heh, i could definitely see that happening. that's why your IDE has a `local history` that can undo time, right? :wink: 
i'll go ahead & take it out again.